fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -8,7 +8,12 @@ function addChild()
 
     var mixin = $("#new_mixin").val();
 
-    var newURI = $('#main').attr('resource') + (!id ? "" : "/" + id);
+    var base = $('#main').attr('resource');
+
+    // Remove trailing '/' if it exists
+    var cleanedBase = base.lastIndexOf('/') == base.length - 1 ? base.substr(0, base.length - 1) : base;
```

Comment:
If you're confident that resource won't end in multiple slashes, we can leave it as-is; otherwise, adding the additional function makes sense to me.